apps/test/netsim/dataConverters.js

```diff
@@ -708,31 +715,31 @@ describe("DataConverters", function () {
      * @param {Base64Payload} right
      */
     var assertBase64NotEqual = function (left, right) {
-      var stringsEqual = (left.string == right.string);
-      var lensEqual = (left.len == right.len);
+      var stringsEqual = (left.string === right.string);
+      var lensEqual = (left.len === right.len);
       assert(!(stringsEqual && lensEqual), "Expected " + JSON.stringify(left) + " and " +
         JSON.stringify(right) + " to be not equal, but they are");
```

Comment:
or removing  this helper entirely and replacing it with calls to `assert.notDeepEqual`